#! /bin/sh
patch -p1 -l -f $* < $0
exit $?

From 78c2361018fdd6e759ef04ebfd2f6f76e104733f Mon Sep 17 00:00:00 2001
From: Andre Heinecke <aheinecke@gnupg.org>
Date: Mon, 19 Dec 2022 13:11:13 +0100
Subject: [PATCH] Update gnupg and libksba

* packages/packages.4 (gnupg): Update.
* packages/packages.common (libksba): Update.
---
 packages/packages.4      | 22 +++++++++++-----------
 packages/packages.common | 10 +++++-----
 2 files changed, 16 insertions(+), 16 deletions(-)

diff --git a/packages/packages.4 b/packages/packages.4
index 457dbdeb..6cef367e 100644
--- a/packages/packages.4
+++ b/packages/packages.4
@@ -23,16 +23,16 @@ server https://gnupg.org/ftp/gcrypt
 file libgcrypt/libgcrypt-1.10.1.tar.bz2
 chk ef14ae546b0084cd84259f61a55e07a38c3b53afc0f546bffcef2f01baffe9de
 
-# last changed 2022-10-14
-# by wk
-# verified: swdb.lst/build_dir
-file gnupg/gnupg-2.3.8.tar.bz2
-chk  540b7a40e57da261fb10ef521a282e0021532a80fd023e75fb71757e8a4969ed
+# last changed 2022-12-19
+# by ah
+# verified: Signature from 6DAA6E64A76D2840571B4902528897B826403ADA
+file gnupg/gnupg-2.4.0.tar.bz2
+chk  1d79158dd01d992431dd2e3facb89fdac97127f89784ea2cb610c600fb0c1483
 
-name gnupg-w32-2.3.8_20221013-bin.exe
-file binary/gnupg-w32-2.3.8_20221013.exe
-chk  ef179c223990f9dfff2b1e6e9f7468908e0d08b381b3abded2471c29faad234e
+name gnupg-w32-2.4.0_20221216-bin.exe
+file binary/gnupg-w32-2.4.0_20221216.exe
+chk  0d4539afefaca0e0a0b0fe0d06baf207512a9520e38d680aaa90c1c57905cf48
 
-name gnupg-w32-2.3.8_20221013-src.tar.xz
-file binary/gnupg-w32-2.3.8_20221013.tar.xz
-chk  f9a4acfd154d094965f689436e4a606717486a367f40f806b5bfb53ae1e513b5
+name gnupg-w32-2.4.0_20221216-src.tar.xz
+file binary/gnupg-w32-2.4.0_20221216.tar.xz
+chk  d9a431699142554bd1ccc3d27d0feb78e006b6d0beef595df8f8c90e08aae7e7
diff --git a/packages/packages.common b/packages/packages.common
index e8f65667..8a88b7eb 100644
--- a/packages/packages.common
+++ b/packages/packages.common
@@ -119,11 +119,11 @@ file npth/npth-1.6.tar.bz2
 chk 1393abd9adcf0762d34798dc34fdcf4d0d22a8410721e76f1e3afcd1daa4e2d1
 
 
-# last changed 2022-10-07
-# by wk
-# verified: build tree
-file libksba/libksba-1.6.2.tar.bz2
-chk  fce01ccac59812bddadffacff017dac2e4762bdb6ebc6ffe06f6ed4f6192c971
+# last changed 2022-12-19
+# by ah
+# verified: Signature from: 6DAA6E64A76D2840571B4902528897B826403ADA
+file libksba/libksba-1.6.3.tar.bz2
+chk  3f72c68db30971ebbf14367527719423f0a4d5f8103fc9f4a1c01a9fa440de5c
 
 
 # last changed 2022-04-07
-- 
2.11.0

